### PR TITLE
Activeadmin

### PR DESCRIPTION
**Pull request by julianpalladino**
**Summary**
- Added active admin. You can now create, edit and delete model instances.

**Screenshots**
screenshot from 2016-02-24 15 21 03
screenshot from 2016-02-24 15 21 21
screenshot from 2016-02-24 15 21 27
screenshot from 2016-02-24 15 21 46

**Trello card**
https://trello.com/c/IS1OdRfe/13-instalar-y-configurar-active-admin

### CODE REVIEW

File: app/admin/comment.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# ActiveAdmin.register Comment do - not sure
+ActiveAdmin.register Comment, :as => "PostComment" do
```
Comment (mdesanti) [inline app/admin/comment.rb]: Why did you add `as: "PostComment"`?

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    div class: "blank_slate_container", id: "dashboard_default_message" do
+      span class: "blank_slate" do
+        span I18n.t("active_admin.dashboard_welcome.welcome")
+        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (mdesanti) [inline app/admin/dashboard.rb]: Delete commented lines

File: app/admin/comment.rb (hunk 1 above)
Comment (julianpalladino) [inline app/admin/comment.rb]: Because Active Admin already used a "comment" model (to comment on resources) and it caused collision issues

Comment (julianpalladino) [inline app/admin/comment.rb]: Should I leave it like that? Or find another way around it?

**Comment by mdesanti** [comment]
@julianpalladino rebase this branch with development please :smile:

File: app/admin/comment.rb (hunk 1 above)
Comment (mdesanti) [inline app/admin/comment.rb]: It's Ok
